src/PriceCalculatorKata/Discount.cs

```diff
@@ -0,0 +1,22 @@
+namespace PriceCalculatorKata;
+public class Discount
+{
+    public static float UniversalDisscount { get; set; }
```

Comment:
I don't think this should be static, I think this class can just be a DiscountService with no static methods.